### ISSUE TITLE
Feature request: v-unless

### ISSUE THREAD

**Issue by b44rd** [vuejs/vue#2691]
While using [rivets.js](http://rivetsjs.com/), I´ve enjoyed a little, nice alternative to **rv-if**, which is **rv-unless**. 

Even though the same can be achieved in vue.js through saying v-if="!property", I believe v-unless="property" could actually make a nice addtion to v-if and v-else. 

In a way, v-unless would actually refer to a v-else block without the v-if. The bonus is you don´t have to check against negated values, which in some cases could make the code a bit harder to read. 

TL;DR: I think v-unless could make a nice addition to vue.js!

**Comment by azamat-sharapov** [comment]
Check out discussion [here](https://github.com/vuejs/vue/pull/1133).

**Comment by b44rd** [comment]
Aha, thanks. I´ll close this feature request then.
